scenarios/kubernetes_bazel_test.py

```diff
@@ -124,14 +124,29 @@ def fake_clean(_dirname, _filename):
     def test_expand(self):
         """Make sure flags are expanded properly."""
         args = kubernetes_bazel.parse_args([
-            '--build=//b/... //c/...'
+            '--build=//b/... -//b/bb/... //c/...'
             ])
         kubernetes_bazel.main(args)
 
         call = self.callstack[-2]
         self.assertIn('//b/...', call)
+        self.assertIn('-//b/bb/...', call)
         self.assertIn('//c/...', call)
 
+
+    def test_query(self):
+        """Make sure query is constructed properly."""
+        args = kubernetes_bazel.parse_args([
+            '--build=//b/... -//b/bb/... //c/...'
+            ])
+        # temporarily un-stub query
+        with Stub(kubernetes_bazel, 'query', self.boiler['query'].old):
+            def check_query(*cmd):
+                self.assertIn('kind(.*_binary, rdeps(//b/... -//b/bb/... +//c/..., //...))', cmd)
+                return '//b/aa/...\n//c/...'
+            with Stub(kubernetes_bazel, 'check_output', check_query):
+                kubernetes_bazel.main(args)
```

Comment:
@ixdy added a "fun" test here that at least asserts that the query we make is as-expected.